wagtail/admin/templates/wagtailadmin/workflows/index.html

```diff
@@ -58,9 +58,17 @@ <h1 class="icon icon-{{ view.header_icon }}">
             </div>
         {% else %}
             {% url view.add_url_name as add_url %}
-            <p>{% blocktrans %}No workflows have been created. Why not <a href="{{ add_url }}">add one</a>?{% endblocktrans %}</p>
+            {% if showing_disabled %}
+                <p>{% blocktrans %}No workflows have been created. Why not <a href="{{ add_url }}">add one</a>?{% endblocktrans %}</p>
+            {% else %}
+                <p>{% blocktrans %}There are no enabled workflows. Why not <a href="{{ add_url }}">add one</a>?{% endblocktrans %}</p>
+            {% endif %}
+        {% endif %}
+        {% if showing_disabled %}
+            <a href="?show_disabled=False" class="button secondary">{% trans "Hide Disabled Workflows" %}</a>
```

Comment:
Standard convention for button text is to only capitalise the first word - "Hide disabled workflows". (It gets capitalised in CSS anyway, but at least this way we keep it consistent for translators...)

Reply:
Thanks, I've fixed this now.